mifosng-android/src/main/res/values/strings.xml

```diff
@@ -751,6 +751,12 @@
         <item>savingsaccounts</item>
     </string-array>
 
+    <string-array name="gender_options_values">
+        <item>Male</item>
+        <item>Female</item>
+        <item>Other</item>
+    </string-array>
```

Comment:
Please first add gender option values as a string ( ` <string name="male">Male</string>` ) then use it inside array items like `<itme>@string/male</item>`. This is going to help us to translate gender options too, otherwise, we can't translate them to other languages which leads to bad user experience.

Reply:
Yes, sure I will do it and thank you for your suggestions :)